introduction.md

```diff
@@ -0,0 +1 @@
+Hello everybody! My name is Pedro Fernandez, and love to work as a tester on agile environments. I am learning git :-)
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.